src/Engine/GoogleCloudVisionEngine.php

```diff
@@ -28,10 +28,11 @@ public function __construct(string $keyFile)
      * Get transcribed text from the given image.
      * @param string $imageUrl
      * @param string[]|null $langs
+     * @param array|null $_extras Not used by the GoogleCloudVision engine.
      * @return string
      * @throws OcrException
      */
-    public function getText(string $imageUrl, ?array $langs = null): string
+    public function getText(string $imageUrl, ?array $langs = null, ?array $_extras = null): string
```

Comment:
Typo underscore in `$_extras`.

Reply:
That was to indicate `$extras` isn't used in this method, we just had to have it as an arg due to the signature in the abstract EngineBase. But, I'll go with your suggestion of having setters instead of an `$extras` arg.